integration-tests/tests/src/tests/list.ts

```diff
@@ -521,6 +522,24 @@ describe("Lists", () => {
   });
   describe("subscripts", () => {
     openRealmBeforeEach({ schema: [LinkTypeSchema, TestObjectSchema, PrimitiveArraysSchema] });
+    it("invalid object access yield correct values", function (this: RealmContext) {
+      this.realm.write(() => {
+        const obj = this.realm.create<ILinkTypeSchema>("LinkTypesObject", {
+          objectCol: { doubleCol: 1 },
+          objectCol1: { doubleCol: 2 },
+          arrayCol: [{ doubleCol: 3 }],
+        });
+
+        //React native returns an empty object upon invalid indexing.
+        if (environment.reactNative) {
+          //@ts-expect-error TYPEBUG: indexing by string on results is not allowed typewise
+          expect(Object.keys(obj?.arrayCol[""]).length).equals(0);
```

Comment:
In my opinion:
- the long term solution is to figure out why this is happening. This behavior shouldn't be platform dependent.
- the short term solution could be creating an issue to track this and disable the test for now, instead of encoding an unexpected behavior as a success.

Reply:
@takameyer was this addressed?

Reply:
Never mind, I updated my UI and found the change I needed to see 👍😅